### ISSUE TITLE
Tell macOS users how to open the terminal

### ISSUE THREAD

**Issue by roberth** [NixOS/nixos-homepage#1116]
**Introduce your stance**

I'm a Nixer first, and I am perpetually new to macOS, as I generally only use it over ssh, when I have to.
That makes launching the Terminal a challenge for me. It would be nice to help out with this.

I have to acknowledge that *macOS users who don't know how to run the Terminal app* is not our target audience, but I'm not a macOS user. I suspect that some other community members fit this profile, but perhaps not all that many.

**Describe the issue**

I'd suggest adding something like this to the download page

    - Open Finder
    - Click the search icon
    - Type Terminal
    - Open it or drag it to the dock first
  
or something that's easier.

**Page links**

https://nixos.org/download#nix-install-macos

**Additional context**

**Comment by camelpunch** [comment]
Cmd-space Terminal?

**Comment by orellazri** [comment]
To be honest I don't think that necessary. Users who plan to use Nix on MacOS should know how to open the terminal (like any other app). It's not Nix's responsibility to explain that.

**Comment by roberth** [comment]
I might be a rare species of user that learns macOS only _because of_ Nix, yet I believe there may be others of my kind.

If not for professional reasons - having to support macOS, they might have just switched to it because of fast hardware, and installing Nix is the first thing they do on the platform.

**Comment by brnl-cfit** [comment]
I'm also very sure the first thing a user would do is press Cmd+Space and type "terminal" or open a browser tab/window and type "open terminal on mac" to have the answer in the (preview of the) first hit.

**Comment by brnl** [comment]
I'm also very sure the first thing a user would do is press Cmd+Space and type "terminal" or open a browser tab/window and type "open terminal on mac" to have the answer in the (preview of the) first hit.

**Comment by roberth** [comment]
I didn't know about Cmd+Space, because I was a Linux user who had to learn the macOS UI for the first time, for work reasons.

If you don't want to help Linux users by spending like 4 words, that seems unfortunate.

**Comment by brnl-cfit** [comment]
Fair point, there's enough room on the page. Maybe something like this?

> **Multi-user installation**
> 
> Install Nix via the recommended [multi-user installation](https://nixos.org/manual/nix/stable/installation/multi-user). Open the terminal (Press Cmd+Space and type 'Terminal') and run:
> 
> >  sh <(curl -L https://nixos.org/nix/install)
> 
> We believe we have ironed out how to cleanly support the read-only root on modern macOS. Please [consult the manual](https://nixos.org/manual/nix/stable/installation/installing-binary#macos-installation) on details what the installation script does.

**Comment by brnl** [comment]
Fair point, there's enough room on the page. Maybe something like this?

> **Multi-user installation**
> 
> Install Nix via the recommended [multi-user installation](https://nixos.org/manual/nix/stable/installation/multi-user). Open the terminal (Press Cmd+Space and type 'Terminal') and run:
> 
> >  sh <(curl -L https://nixos.org/nix/install)
> 
> We believe we have ironed out how to cleanly support the read-only root on modern macOS. Please [consult the manual](https://nixos.org/manual/nix/stable/installation/installing-binary#macos-installation) on details what the installation script does.

**Comment by roberth** [comment]
Yeah, that's good. I think we can even recover some brevity.

I do agree with the earlier idea that we can assume some level of familiarity:
- with GUI instructions, such as "+" syntax
- basic terminal/shell skill level, e.g. no need to say "and hit enter"

My suggestion would be:

> **Multi-user installation**
> Install Nix via the recommended [multi-user installation](https://nixos.org/manual/nix/stable/installation/multi-user). Run in the terminal ([Cmd]+[Space] and type `Terminal`):
> > sh <(curl -L https://nixos.org/nix/install)

**Comment by clemens-tolboom** [comment]
I am a MacOS user for long so know [Cmd]-[Space] but I ran the script as `sudo` ... hence my PR #1116

I totally agree to add `[Cmd]-[Space]` as some of my co-worker are new to MacOS
